tests/unit/test_session_legacy.py

```diff
@@ -708,7 +708,10 @@ def test_create_client_verify_param_overrides_all(self, client_creator):
             self.assertEqual(call_kwargs['verify'], 'verify-certs.pem')
 
     @mock.patch('botocore.client.ClientCreator')
-    def test_create_client_use_no_api_version_by_default(self, client_creator):
+    def test_create_client_use_no_api_version_by_default(
```

Comment:
Yeah same thing here. I would have not expected this file to change unless we were updating logic in the tests.

Reply:
From previous refactors this line was touched and then `pre-commit` cleaned it up. Will revert it.